src/components/Pages/Catalog/index.scss

```diff
@@ -0,0 +1,77 @@
+@import '~antd/dist/antd.css';
+
+.filter-wrapper {
+  width: 100%;
+  height: auto;
+  display: flex;
+  justify-content: center;
+  margin: 15px 0;
+  &__footer {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    width: 100%;
+    &-button {
+      color: rgb(156, 156, 156);
+      font-size: 1.2em;
+      padding: 5px;
+      margin: 5px 10px;
+      max-width: max-content;
+      border-radius: 4px;
+    }
+  }
+}
+
+.filter-wrapper__footer-button:hover {
+  color: #ffffff;
+}
+
+.filter-wrapper__container {
+  border-radius: 15px;
+  width: 100%;
+  max-width: max-content;
+  background-color: rgba(31,27,46,.88);
```

Comment:
would be worth to put the colors to the variables, e.g - `$base-color: rgba(31,27,46,.88);` 
And please, use RGB or HEX